setup.py

```diff
@@ -25,10 +25,13 @@
 # override these and pass --static for a static build. See
 # doc/build.txt for more information. If you do not pass --static
 # changing this will have no effect.
-STATIC_INCLUDE_DIRS = []
-STATIC_LIBRARY_DIRS = []
-STATIC_CFLAGS = []
-STATIC_BINARIES = []
+def static_env_list(name):
+    return [x.strip() for x in os.environ.get(name, "").split("|") if x.strip()]
```

Comment:
```suggestion
    return [x.strip() for x in os.environ.get(name, "").split(",") if x.strip()]
```

Reply:
I thought of using `|` instead of `,` because compiler _CFLAGS_ usually rely on the comma.  For example, when passing linker flags through the front end, e.g. `-Wl,-s`.

Reply:
Ah, sorry, I considered that an accidental edit artefact.
So, add an additional `separator` argument to the function and make it `';'` for path values and spaces (i.e. `None`) for CFLAGS?

Reply:
Sure.  Instead of `;`, would it be better to use [`os.pathsep`][1]?

[1]: https://docs.python.org/3.9/library/os.html#os.pathsep

Reply:
Absolutely :)